opm/autodiff/BlackoilModelBase.hpp

```diff
@@ -259,6 +259,12 @@ namespace Opm {
         WellModel& wellModel() { return well_model_; }
         const WellModel& wellModel() const { return well_model_; }
 
+        /// Calculate FIP
```

Comment:
It should be documented what is returned, what is the size and content of the returned object.